test/verify/check-docker

```diff
@@ -25,6 +25,20 @@ import os
 
 @skipImage("No cockpit-docker on i386", "fedora-i386")
 class TestDocker(MachineCase):
+    def setUp(self):
+        MachineCase.setUp(self)
+        m = self.machine
+
+        # HACK: https://github.com/docker/docker/issues/25246
+        if m.image in [ "debian-8", "debian-testing" ]:
+            m.execute("printf '\\n\\n[Service]\\nKillMode=process\\n' >> /lib/systemd/system/docker.service")
+            m.execute("systemctl daemon-reload")
+
+        m.execute("systemctl start docker")
+
+        # HACK: docker fails to start a container without /etc/resolv.conf
+        m.execute("touch /etc/resolv.conf")
```

Comment:
This is unlikely to be solved in Docker proper for a while. Given the age of the bug. But I guess you guys need to decide which way to go here. I've already spent way too many hours on this on doing it two different ways, and at least half of that work (if not all) will be thrown away.